src/libs/OptionsListUtils.js

```diff
@@ -432,11 +432,37 @@ function getHeaderMessage(hasSelectableOptions, hasUserToInvite, searchValue, ma
     return '';
 }
 
+/**
+ * Returns the personal details for an array of logins
+ *
+ * @param {Object} currencyListObject
+ * @param {String} searchValue
+ * @param {Object} selectedCurrency
+ * @returns {Array}
+ */
+function getCurrencyListForSections(currencyListObject, searchValue) {
+    const currencyListKeys = _.keys(currencyListObject);
+    const currencyOptions = _.map(currencyListKeys, currencyCode => ({
+        text: `${currencyCode} - ${currencyListObject[currencyCode].symbol}`,
+        searchText: `${currencyCode} ${currencyListObject[currencyCode].symbol}`,
+        currencyCode,
+    }));
+
+    const filteredOptions = currencyOptions.filter(currencyOption => (
+        isSearchStringMatch(searchValue, currencyOption.searchText)));
+
+    return {
+        // currency options holds a section for those currencies which are not selected
```

Comment:
I'm not sure what this comment means, could you please clarify?

Reply:
Sorry @Julesssss  I think this was missed while I was staging changes. I will change it.